livedata/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -89,25 +37,24 @@ class EventHandlerImpl(var domainImpl: ChatDomainImpl, var runAsync: Boolean = t
         val channelsToFetch = mutableSetOf<String>()
         val messagesToFetch = mutableSetOf<String>()
 
+        events.filterIsInstance<CidEvent>().onEach { channelsToFetch += it.cid }
+
         // step 1. see which data we need to retrieve from offline storage
         for (event in events) {
             when (event) {
-                is MessageReadEvent -> channelsToFetch += event.cid
-                is MemberAddedEvent -> channelsToFetch += event.cid
-                is MemberRemovedEvent -> channelsToFetch += event.cid
-                is NotificationRemovedFromChannelEvent -> channelsToFetch += event.cid
-                is MemberUpdatedEvent -> channelsToFetch += event.cid
-                is ChannelUpdatedEvent -> channelsToFetch += event.cid
-                is ChannelDeletedEvent -> channelsToFetch += event.cid
-                is ChannelHiddenEvent -> channelsToFetch += event.cid
-                is ChannelVisibleEvent -> channelsToFetch += event.cid
-                is NotificationAddedToChannelEvent -> channelsToFetch += event.cid
-                is NotificationInvitedEvent -> channelsToFetch += event.cid
-                is NotificationInviteAcceptedEvent -> channelsToFetch += event.cid
-                is ChannelTruncatedEvent -> channelsToFetch += event.cid
+                is MessageReadEvent, is MemberAddedEvent, is MemberRemovedEvent, is NotificationRemovedFromChannelEvent,
+                is MemberUpdatedEvent, is ChannelUpdatedEvent, is ChannelDeletedEvent, is ChannelHiddenEvent,
+                is ChannelVisibleEvent, is NotificationAddedToChannelEvent, is NotificationInvitedEvent,
+                is NotificationInviteAcceptedEvent, is ChannelTruncatedEvent, is ChannelCreatedEvent, is HealthEvent,
+                is NotificationMutesUpdatedEvent, is GlobalUserBannedEvent, is UserDeletedEvent, is UserMutedEvent,
+                is UsersMutedEvent, is UserPresenceChangedEvent, is GlobalUserUnbannedEvent, is UserUnmutedEvent,
+                is UsersUnmutedEvent, is UserUpdatedEvent, is NotificationChannelMutesUpdatedEvent, is ConnectedEvent,
+                is ConnectingEvent, is DisconnectedEvent, is ErrorEvent, is UnknownEvent, is NotificationMessageNewEvent,
+                is NotificationChannelDeletedEvent, is NotificationChannelTruncatedEvent, is NotificationMarkReadEvent,
+                is TypingStartEvent, is TypingStopEvent, is ChannelUserBannedEvent, is UserStartWatchingEvent,
+                is UserStopWatchingEvent, is ChannelUserUnbannedEvent -> Unit
```

Comment:
It is good to group all "empty cases" into one statement, but, could you keep them in different lines?
It is easier to read and will help us minimizing conflict on futures changes

Reply:
fix